Atmosphere/index.html

```diff
@@ -0,0 +1,115 @@
+             <h4>magna</h4>
+             <h2>Dolor sit amet vivamus</h2>
+             <p>Phasellus in ante vitae nibh porttitor tempus vitae ut ante. Vestibulum blandit ex a elementum viverra. Sed quat diam, aliquet tempus felis nisl at lacus.</p>
+             <a href="#" class="btn-one">more</a>
+            </section>
+            <section class="card">
+           <h4>LOREM</h4>
+            <h2>Ipsum dolor sit amet</h2>
+            <p>Phasellus in ante vitae nibh porttitor tempus vitae ut ante. Vestibulum blandit ex a elementum viverra. Sed quat diam, aliquet tempus felis nisl at lacus.</p>
+            <a href="#" class="btn-one">more</a>   
+            </section>
+        </article>
+        <article class="description">
+            <section>
+                <div>
+                    <i class="fa-solid fa-chart-column post-icons"></i>
+                </div>
+               <div>
+                <h4>PLACERAT</h4>
+                <P>Lorem ipsum dolor sit amet.nullam et consequate</P>
+               </div>
+               <div>
+                <i class="fa-brands fa-telegram post-icons"></i>
+                 <div>
+                  <h4>LIBERO</h4>
+                  <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+                 <div>
+                    <i class="fa-solid fa-chart-area post-icons"></i>
+                    
+                 </div>
+                 <div>
+                    <h4>SOLICITU</h4>
+                    <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+                 <div>
+                    <i class="fa-solid fa-image post-icons"></i> 
+                 </div>
+                 <div>
+                    <h4>TEMPOR</h4>
+                    <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+            </section>
+        </article>
+
+        <article class="img-article">
+            <section class="class-img">
+              <h4>CONSEQUAT</h4>
+              <p>Fusce lobortis lorem semper sagittis</p><hr>
+            </section>
+            <section>
+                <img src="assest/pic01.jpg" alt="" class="man-img">
+            </section>
+             <section class="class-img">
+                <p>Phasellus in ante vitae nibh porttitor tempus vitae ante. Lorem blandit dolor elementum dolor. Sed quat diam, aliquet tempus felis. Phasellus et magna vitae sed porttitor tempus vitae ut ante lorem blandit.</p>
+                <a href="#" class="btn-one">more</a>
+             </section>
+        </article>
+        
+           <article class="aliquam-decription">
+            <section class="aliquam">
+                <h1>MALESUADA</h1>
+                <p>Aliquam tincidunt mauris eu risus</p>
+            </section>
+            <section class="aliquam">
+                <h1>09.04.16</h1>
+                <p>MAGNA ETIAM SED LOREM IPSUM DOLOR AMET</p>
+            </section>
+            <section class="aliquam">
```

Comment:
maring is too much check for and font sizing and correct the font color spacing etc